src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -46,8 +47,13 @@ function ExamSettings(props) {
 
     if (provider === 'proctortrack') {
       setCreateZendeskTickets(false);
-    } else if (provider === 'software_secure') {
-      setCreateZendeskTickets(true);
+      setShowProctortrackEscalationEmail(true);
+    } else {
+      if (provider === 'software_secure') {
+        setCreateZendeskTickets(true);
+      }
+      setProctortrackEscalationEmail('');
```

Comment:
I think this behavior is different than what is happening on the Advanced Settings page currently. If a user switches from proctortrack to another provider, the escalation email field doesn't change automatically. I'm not sure if we want to exactly match what is happening on Advanced Settings, but if we are already hiding this field on the MFE is there a need to change the escalation email as well?

Reply:
It's a good product question. We can talk about this in our next meeting. I'll bring this up to the team to decide.

Reply:
Ok, took in your feedback. Thank you.